sharepoint/sharepoint-ps/sharepoint-online/Get-SPOAppBillingPolicies.md

```diff
@@ -0,0 +1,145 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spocontainer
+applicable: SharePoint Online
+title: Get-SPOAppBillingPolicies
+schema: 2.0.0
+author: arakesh
+ms.author: arakesh
+ms.reviewer:
+---
+
+# Get-SPOAppBillingPolicies
+
+## SYNOPSIS
+
+Returns billing policies that are owned by the tenant.
+
+## SYNTAX
+
+```powershell
+
+Get-SPOAppBillingPolicies

```

Comment:
I think you also need to add the parameters here as well.

Reply:
There are no parameters to input in Get - and I have added the parameters that are an output of this cmdlet. Let me know if I am missing something.

Reply:
I think the PARAMETERS section actually refers to the inputs, not the outputs

I see some files have an "Output" section, but I don't see any good examples of how that should look like